### ISSUE TITLE
dotenvx get MY_VAR doesn't show anything

### ISSUE THREAD

**Issue by pfraces-rc** [dotenvx/dotenvx#386]
Since version `1.6.0` the `get` command doesn't show anything to the stdout

**Steps to reproduce:**

```
mkdir test
cd test
npm init -y
npm install @dotenvx/dotenvx
touch .env
npx dotenvx set FOO bar --plain
npx dotenvx get FOO
```

**Expected result:**

- `bar` is shown

**Actual result:**

- Nothing is shown

**More info:**

- It also happens with encrypted .env files
- I've tested several versions and the last version working is `1.5.0`

**Comment by motdotla** [comment]
interesting. only happening with npx for some reason. i'll investigate.

until then install the cli as a binary or place in a run script.

**Comment by motdotla** [comment]
i moved from logger blank to `process.stdout` so must be related:

https://github.com/dotenvx/dotenvx/compare/v1.5.0...v1.6.0#diff-e70d88604367fe48aaeea73514ff16488aa722799bc4efcbc39b4a2312248f3cR29

**Comment by motdotla** [comment]
from quick research i think moving to console.log (and i'll just chomp the newline) will fix things here. thank you for catching this!

**Comment by motdotla** [comment]
weird. if you output that as stdout to a file it is not truncated:

```
$ npx dotenvx get FOO > file.txt
$ echo file.txt
bar
```

**Comment by motdotla** [comment]
also if you run:

```
npx @dotenvx/dotenvx get -pp
```

you will get everything, except the last line. `npx` seems to truncate it unless it ends with a newline. wtf.

**Comment by motdotla** [comment]
patched in `1.14.2` and greater: https://github.com/dotenvx/dotenvx/pull/387

`dotenvx get --format=shell` also still works correctly. was a bit worried about that one. we're all good. 🤙
